install/scripts/install-rocm.sh

```diff
@@ -6,6 +6,9 @@ rm -rf $BUILD_DIR
 
 #git clone -b hipsycl-0.8 https://github.com/illuhad/aomp $BUILD_DIR/aomp
 git clone https://github.com/ROCm-Developer-Tools/aomp $BUILD_DIR/aomp
+cd $BUILD_DIR/aomp
+git fetch --all
+git checkout rel_0.7-7
```

Comment:
We also may want to have a variable for this in the future.

Reply:
Sure thing, I can add this to this commit right away. Is there a reason why we would want this done in a separate commit?

Reply:
You can do this now if you prefer - see my comment above.